README.adoc

```diff
@@ -21,39 +21,52 @@ You can install the gem using:
 
  $ gem install asciidoctor
 
-If you are using Bundler to manage the dependencies in your Jekyll project, then add the Asciidoctor gem instead to your `Gemfile` (below the Jekyll gem, named jekyll):
+If you are using Bundler to manage the dependencies in your Jekyll project, then add the Asciidoctor gem to your `Gemfile` (below the Jekyll gem, named jekyll):
 
 [source,ruby]
+----
 source 'https://rubygems.org'
+
 gem 'jekyll'
 gem 'asciidoctor'
+----
 
-Then, run the Bundler install command, `bundle`:
-
- $ bundle
-
-=== Using a release
-
-First, install the Jekyll AsciiDoc gem (named `jekyll-asciidoc`):
+Then, run the Bundler install command
 
- $ gem install jekyll-asciidoc`
+ $ bundle install
 
-If you are using Bundler to manage the dependencies, add the `jekyll-asciidoc` gem to your `Gemfile`:
+=== Installing release version of the plugin
 
+* Using Bundler
```

Comment:
"Using Bundler" and "Without Bundler" might work better as a description list. Something like:

```
Using Bundler::
// content

Without Bundler::
// content
```